File: src/main/java/com/my/admin/app/commerce/model/document/ProductTopping.java
package com.my.admin.app.commerce.model.document;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.io.Serializable;

@Document(collection = "product_topping")
@Data
public class ProductTopping implements Serializable {

    private static final long serialVersionUID = 1L;

    @Id
    private Long mainDishId;

    @Id
    private Long toppingId;
}

File: src/main/kotlin/com/bk/reactive/app/commerce/admin/my/model/document/ProductTopping.kt
package com.bk.reactive.app.commerce.admin.my.model.document

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document

@Document(collection = "product_topping")
data class ProductTopping(
    @Id
    val id: String,
    val mainDishId: Long,
    val toppingId: Long
)